baselines/naming/samples/generated/v1beta1/naming.paginated_method_async.js.baseline

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+'use strict';
+
+function main() {
+  // [START naming_paginated_method_async_sample]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+
+  // Imports the Naming library
+  const {NamingClient} = require('naming').v1beta1;
+
+  // Instantiates a client
+  const namingClient = new NamingClient();
+
+  async function paginatedMethodAsync() {
+    // Construct request
+    const request = {
+    };
+
+    // Run request
+    const response = await namingClient.paginatedMethodAsync(request);
```

Comment:
For `async` paginated examples, I think we should show:

```js
for async (const resp of await namingClient.paginatedMethodAsync(request)) {
  console.info(resp);
}
```

That way folks understand the preferred way to paginate over lists.

Reply:
Great point.

Reply:
Hey! So I couldn't figure out where these protos are in IRL, but as far as I know, this is happening in other libraries that have async methods. See, for example:

```  // Imports the Asset library
  const {AssetServiceClient} = require('asset').v1;

  // Instantiates a client
  const assetClient = new AssetServiceClient();

  async function listAssets() {
    // Construct request
    const request = {
      parent,
    };

    // Run request
    const iterable = await assetClient.listAssetsAsync(request);
    for await (const response of iterable) {
        console.log(response);
    }
  }

  listAssets();
  // [END asset_list_assets_sample]
}

process.on('unhandledRejection', err => {
  console.error(err.message);
  process.exitCode = 1;
});
main(...process.argv.slice(2));```

Is this a real API, if so, can I try to test it locally? Otherwise, I'd be tempted to say it's not actually listed as a service.paging method?

Reply:
Checked on https://github.com/googleapis/googleapis/blob/master/google/cloud/asset/v1/asset_service.proto
The `listFeed` is not following the [AIP-132](https://google.aip.dev/132). So it is unary call. That's why it doesn't have `Async` method.